src/infra/main/template.yaml

```diff
@@ -452,6 +448,12 @@ Resources:
       PolicyDocument:
         Version: "2012-10-17"
         Statement:
+          - Effect: Allow
```

Comment:
Hi @chiarapascucci-gds - I think this creates a issue when deploying as the role upon creation does not have access to the queue and therefore cannot create the trigger hence why its an inline policy in this context.  If you add a dependson to the role then a circular dependency is created - I remember Jess saying it had to be done this way

Reply:
Is this an issue only when deploying through secure pipelines? I deployed this version fine as a manual deployment with my credentials